tools/proto_format/proto_sync.py

```diff
@@ -239,8 +239,12 @@ def get_previous_message_type_deps(proto_path):
     matches = re.findall(PREVIOUS_MESSAGE_TYPE_REGEX, contents)
     deps = []
     for m in matches:
-        target = '//%s:pkg' % get_directory_from_package(m)
-        deps.append(target)
+        # HACK
+        IGNORED = ["//envoy/config/filter/http/squash/v2:pkg"]
+        pkg = get_directory_from_package(m)
```

Comment:
@alyssawilk i tested out your (closed) PR  (#16336 ) here

it would be good if we can keep the previous_message_type annotations as they can be used for pointing to old docs (its a `string` so should be doable i think)

keeping it but removing the dep on the v2 proto required this hack - or something like it

Reply:
oh interesting.  Are you sure it'll point at old docs correctly if the docs aren't in the same release version of Envoy?  If so I'd ping @htuch or @adisuissa on the best way to go forward here.  I think not so bad if at the end of v2 removal we can just ignore *v2:pkg"

Reply:
yep - we have `intersphinx` now for versioning to do just that

Reply:
i have a strong opinion that including the old protos is v important - from bitter experience - trying to find the equivs is a major pain point

im working on a PR that will make use of these annos with  intersphinx and resolve this - once it is in place users will be able to search with the old config name and find the new version (with links back to old for comparison)